localstack/dns/server.py

```diff
@@ -695,6 +695,48 @@ def clear(self):
         self.resolver.clear()
 
 
+class SeparateProcessDNSServer(Server, DnsServerProtocol):
+    def __init__(
+        self,
+        port: int,
+        host: str = "0.0.0.0",
+    ) -> None:
+        super().__init__(port, host)
+
+    @property
+    def protocol(self):
+        return "udp"
```

Comment:
Strictly not true, right? We accept TCP and UDP. Not that it matters, since the protocol is only used for the url which we don't use anyway.